packages/vue/src/components/organisms/SfSidebar/SfSidebar.stories.js

```diff
@@ -241,6 +254,7 @@ const Template = (args, { argTypes }) => ({
     :overlay="overlay"
     :class="classes"
```

Comment:
I think this class prop is not needed anymore wdyt?
You can also remove `classes` argType from 98-107 lines.

Reply:
make sense, done :)